framework/src/modules/chain/bft/bft.js

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const assert = require('assert');
+const { HeadersList } = require('./headers_list');
+const blockHeaderSchema = require('./block_header_schema');
+const { validate } = require('../../../../src/controller/validator');
+
+/**
+ * Validate schema of block header
+ *
+ * @param {BlockHeader} blockHeader
+ * @return {boolean}
+ */
+const validateBlockHeader = blockHeader =>
+	validate(blockHeaderSchema, blockHeader);
```

Comment:
we can call validate directly since it doesn't have any logic inside

Reply:
For now its just schema validation, but its nice to keep it a separate function to later look into any dynamic validation that can't be covered by schema.